udata/api/commands.py

```diff
@@ -59,3 +63,23 @@ def validate():
         success('API specifications are valid')
     except schemas.SchemaValidationError as e:
         exit_with_error('API specifications are not valid', e)
+
+
+@grp.command()
+@click.option('-u', '--user-email', help='User\'s email')
+@click.option('--uri', default='http://localhost:8080/login', help='Client\'s redirect uri')
```

Comment:
suggestion about the uri declaration from the cli : allow declaring several urls by passing them as follow ? 

```@click.option('--uri', default='http://localhost:8080/login+http://localhost:8080', help='Client\'s redirect uri')```

Reply:
or something like (cf [click docs](https://click.palletsprojects.com/en/7.x/options/#multiple-options):
```@click.option('--uri', default='http://localhost:8080/login', help='Client\'s redirect uri', multiple=True)```

Reply:
`default='http://localhost:8080/login+http://localhost:8080'` not sure what kind of syntax this is looks very JS to me 😄 
`default=['http://localhost:8080/login']` will not work click is not designed for this it would need to implement a custom argument type.
I will use `'http://localhost:8080/login http://localhost:8081 http://localhost:8082'` as it is easy to split a string by whitespace.
`default='http://localhost:8080/login+http://localhost:8080'` seems likewise but not sure what kind of syntax this is,  looks very JS to me 😄

Reply:
Btw you already have a comment for this just above

Reply:
https://click.palletsprojects.com/en/7.x/options/#multiple-options ?

Reply:
(like Julien suggested actually)